frontend/src/component/admin/groups/GroupForm/GroupForm.tsx

```diff
@@ -108,18 +134,46 @@ export const GroupForm: FC<IGroupForm> = ({
                     data-testid={UG_DESC_ID}
                 />
                 <ConditionallyRender
-                    condition={Boolean(uiConfig.flags.syncSSOGroups)}
+                    condition={
+                        Boolean(uiConfig.flags.syncSSOGroups) && !isOss()
+                    }
                     show={
-                        <>
-                            <StyledInputDescription>
-                                Is this group associated with SSO groups?
-                            </StyledInputDescription>
-                            <StyledItemList
-                                label="SSO group ID / name"
-                                value={mappingsSSO}
-                                onChange={setMappingsSSO}
-                            />
-                        </>
+                        <ConditionallyRender
+                            condition={Boolean(isGroupSyncingEnabled)}
+                            show={
+                                <>
+                                    <StyledInputDescription>
+                                        Is this group associated with SSO
+                                        groups?
+                                    </StyledInputDescription>
+                                    <StyledItemList
+                                        label="SSO group ID / name"
+                                        value={mappingsSSO}
+                                        onChange={setMappingsSSO}
+                                    />
+                                </>
+                            }
+                            elseShow={() => (
+                                <StyledDescriptionBlock>
+                                    <div>
+                                        You can enable SSO groups syncronization
+                                        if needed
+                                        <Tooltip
+                                            title="You can enable SSO groups
+                                            syncronization if needed"
+                                            arrow
+                                        >
+                                            <StyledHelpOutline />
+                                        </Tooltip>
+                                    </div>
+                                    <Link data-loading to={`/admin/auth`}>
+                                        <span data-loading>
+                                            View SSO configuration{' '}
```

Comment:
Unneeded extra space?
```suggestion
                                            View SSO configuration
```